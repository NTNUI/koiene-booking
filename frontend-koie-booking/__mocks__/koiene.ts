import AdminBooking, { AdminBookingDictionary } from '../src/types/admin/AdminBooking';

export default function getKoieData(): AdminBookingDictionary {
  return koieData;
}

const koieData: AdminBookingDictionary = {
  ['flåkoia']: {
    name: 'Flåkoia',
    slug: 'flåkoia',
    numberOfBeds: 11,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 0,
      ['2020-09-16']: 0,
      ['2020-09-17']: 0,
      ['2020-09-18']: 11,
      ['2020-09-19']: 9,
      ['2020-09-20']: 0,
      ['2020-09-21']: 11,
      ['2020-09-22']: 11,
      ['2020-09-23']: 11,
      ['2020-09-24']: 11,
    },
  },
  ['mevasskoia']: {
    name: 'Mevasskoia',
    slug: 'mevasskoia',
    numberOfBeds: 5,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 5,
      ['2020-09-16']: 5,
      ['2020-09-17']: 5,
      ['2020-09-18']: 1,
      ['2020-09-19']: 0,
      ['2020-09-20']: 3,
      ['2020-09-21']: 5,
      ['2020-09-22']: 2,
      ['2020-09-23']: 5,
      ['2020-09-24']: 5,
    },
  },
  ['øvensenget']: {
    name: 'Øvensenget',
    slug: 'øvensenget',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['kåsen']: {
    name: 'Kåsen',
    slug: 'kåsen',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['stakkslettbua']: {
    name: 'Stakkslettbua',
    slug: 'stakkslettbua',
    numberOfBeds: 11,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 11,
      ['2020-09-16']: 11,
      ['2020-09-17']: 11,
      ['2020-09-18']: 11,
      ['2020-09-19']: 11,
      ['2020-09-20']: 11,
      ['2020-09-21']: 11,
      ['2020-09-22']: 11,
      ['2020-09-23']: 11,
      ['2020-09-24']: 11,
    },
  },
  ['holmsåkoia']: {
    name: 'Holmsåkoia',
    slug: 'holmsåkoia',
    numberOfBeds: 20,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 20,
      ['2020-09-16']: 20,
      ['2020-09-17']: 20,
      ['2020-09-18']: 20,
      ['2020-09-19']: 20,
      ['2020-09-20']: 20,
      ['2020-09-21']: 20,
      ['2020-09-22']: 20,
      ['2020-09-23']: 20,
      ['2020-09-24']: 20,
    },
  },
  ['sonvasskoia']: {
    name: 'Sonvasskoia',
    slug: 'sonvasskoia',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['nicokoia']: {
    name: 'Nicokoia',
    slug: 'nicokoia',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['telin']: {
    name: 'Telin',
    slug: 'telin',
    numberOfBeds: 9,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 9,
      ['2020-09-16']: 9,
      ['2020-09-17']: 9,
      ['2020-09-18']: 9,
      ['2020-09-19']: 9,
      ['2020-09-20']: 9,
      ['2020-09-21']: 9,
      ['2020-09-22']: 9,
      ['2020-09-23']: 9,
      ['2020-09-24']: 9,
    },
  },
  ['lyngli']: {
    name: 'Lyngli',
    slug: 'lyngli',
    numberOfBeds: 13,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 13,
      ['2020-09-16']: 13,
      ['2020-09-17']: 13,
      ['2020-09-18']: 13,
      ['2020-09-19']: 13,
      ['2020-09-20']: 13,
      ['2020-09-21']: 13,
      ['2020-09-22']: 13,
      ['2020-09-23']: 13,
      ['2020-09-24']: 13,
    },
  },
  ['kråklikåten']: {
    name: 'Kråklikåten',
    slug: 'kråklikåten',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['heinfjordstua']: {
    name: 'Heinfjordstua',
    slug: 'heinfjordstua',
    numberOfBeds: 25,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 25,
      ['2020-09-16']: 25,
      ['2020-09-17']: 25,
      ['2020-09-18']: 25,
      ['2020-09-19']: 25,
      ['2020-09-20']: 25,
      ['2020-09-21']: 25,
      ['2020-09-22']: 25,
      ['2020-09-23']: 25,
      ['2020-09-24']: 25,
    },
  },
  ['høgnabu']: {
    name: 'Høgnabu',
    slug: 'høgnabu',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['stabburet']: {
    name: 'Stabburet',
    slug: 'stabburet',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
  ['vekvessætra']: {
    name: 'Vekvessætra',
    slug: 'vekvessætra',
    numberOfBeds: 20,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 20,
      ['2020-09-16']: 20,
      ['2020-09-17']: 20,
      ['2020-09-18']: 20,
      ['2020-09-19']: 20,
      ['2020-09-20']: 20,
      ['2020-09-21']: 20,
      ['2020-09-22']: 20,
      ['2020-09-23']: 20,
      ['2020-09-24']: 20,
    },
  },
  ['fosenkoia']: {
    name: 'Fosenkoia',
    slug: 'fosenkoia',
    numberOfBeds: 10,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 10,
      ['2020-09-16']: 10,
      ['2020-09-17']: 10,
      ['2020-09-18']: 10,
      ['2020-09-19']: 10,
      ['2020-09-20']: 10,
      ['2020-09-21']: 10,
      ['2020-09-22']: 10,
      ['2020-09-23']: 10,
      ['2020-09-24']: 10,
    },
  },
  ['tågåbu']: {
    name: 'Tågåbu',
    slug: 'tågåbu',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['lynhøgen']: {
    name: 'Lynhøgen',
    slug: 'lynhøgen',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['selbukåten']: {
    name: 'Selbukåten',
    slug: 'selbukåten',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
  ['holvassgamma']: {
    name: 'Holvassgamma',
    slug: 'holvassgamma',
    numberOfBeds: 8,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 8,
      ['2020-09-16']: 8,
      ['2020-09-17']: 8,
      ['2020-09-18']: 8,
      ['2020-09-19']: 8,
      ['2020-09-20']: 8,
      ['2020-09-21']: 8,
      ['2020-09-22']: 8,
      ['2020-09-23']: 8,
      ['2020-09-24']: 8,
    },
  },
  ['kamtjønnkoia']: {
    name: 'Kamtjønnkoia',
    slug: 'kamtjønnkoia',
    numberOfBeds: 6,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 6,
      ['2020-09-16']: 6,
      ['2020-09-17']: 6,
      ['2020-09-18']: 6,
      ['2020-09-19']: 6,
      ['2020-09-20']: 6,
      ['2020-09-21']: 6,
      ['2020-09-22']: 6,
      ['2020-09-23']: 6,
      ['2020-09-24']: 6,
    },
  },
  ['rindalsløa']: {
    name: 'Rindalsløa',
    slug: 'rindalsløa',
    numberOfBeds: 4,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 4,
      ['2020-09-16']: 4,
      ['2020-09-17']: 4,
      ['2020-09-18']: 4,
      ['2020-09-19']: 4,
      ['2020-09-20']: 4,
      ['2020-09-21']: 4,
      ['2020-09-22']: 4,
      ['2020-09-23']: 4,
      ['2020-09-24']: 4,
    },
  },
  ['iglbu']: {
    name: 'Iglbu',
    slug: 'iglbu',
    numberOfBeds: 10,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 10,
      ['2020-09-16']: 10,
      ['2020-09-17']: 10,
      ['2020-09-18']: 10,
      ['2020-09-19']: 10,
      ['2020-09-20']: 10,
      ['2020-09-21']: 10,
      ['2020-09-22']: 10,
      ['2020-09-23']: 10,
      ['2020-09-24']: 10,
    },
  },
  ['mortenskåten']: {
    name: 'Mortenskåten',
    slug: 'mortenskåten',
    numberOfBeds: 2,
    bedsAvailableInDateRange: {
      ['2020-09-15']: 2,
      ['2020-09-16']: 2,
      ['2020-09-17']: 2,
      ['2020-09-18']: 2,
      ['2020-09-19']: 2,
      ['2020-09-20']: 2,
      ['2020-09-21']: 2,
      ['2020-09-22']: 2,
      ['2020-09-23']: 2,
      ['2020-09-24']: 2,
    },
  },
};
