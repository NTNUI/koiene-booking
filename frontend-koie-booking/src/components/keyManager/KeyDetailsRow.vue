<template>
  <tr style="background-color: #222222;">
    <td>
      {{ keyDetail.email }}
    </td>
    <td>
      {{ keyDetail.koieName }}
    </td>
    <td>
      <span v-if="isPickup" :style="getColorFn(keyDetail.startDate)">{{ keyDetail.startDate }} </span>
      <span v-else> {{ keyDetail.startDate }} </span>
    </td>
    <td>
      <span v-if="!isPickup" :style="getColorFn(keyDetail.startDate)">{{ keyDetail.endDate }} </span>
      <span v-else> {{ keyDetail.endDate }} </span>
    </td>
    <td>
      {{ keyDetail.status }}
    </td>
  </tr>
</template>

<script lang="ts">
import Vue from 'vue';
import keyTableHeaders from '@/components/keyManager/keyTableHeaders';
import KeyDetail from '../../types/keyManager/KeyDetail';
import dayjs from 'dayjs';
import scssVars from '@/styles/variables.scss';

export default Vue.extend({
  name: 'KeyPickUpRow',
  props: {
    keyDetail: {
      type: Object as () => KeyDetail,
      default() {
        return null;
      },
    },
    getColorFn: {
      type: Function,
      default: (startDate: string) => Object,
    },
    isPickup: {
      type: Boolean,
      default: true,
    },
  },
  data() {
    return {
      headers: keyTableHeaders,
    };
  },
});
</script>

<style scoped></style>
