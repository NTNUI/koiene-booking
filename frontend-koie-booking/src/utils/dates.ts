import dayjs from 'dayjs';
require('dayjs/locale/nb');
dayjs.locale('nb');

export function getDateString(baseDate?: string, addDays?: number): string {
  let resultString: string;
  if (baseDate && addDays) {
    resultString = new Date(new Date(baseDate).getTime() + addDays * 24 * 60 * 60 * 1000).toISOString().substr(0, 10);
  } else if (baseDate) {
    resultString = new Date(baseDate).toISOString().substr(0, 10);
  } else if (addDays) {
    resultString = new Date(new Date().getTime() + addDays * 24 * 60 * 60 * 1000).toISOString().substr(0, 10);
  } else {
    resultString = new Date().toISOString().substr(0, 10);
  }
  return resultString;
}

export function getDate(baseDate?: string, addDays?: number): Date {
  let resultDate: Date;
  if (baseDate && addDays) {
    resultDate = new Date(new Date(baseDate).getTime() + addDays * 24 * 60 * 60 * 1000);
  } else if (baseDate) {
    resultDate = new Date(baseDate);
  } else if (addDays) {
    resultDate = new Date(new Date().getTime() + addDays * 24 * 60 * 60 * 1000);
  } else {
    resultDate = new Date();
  }
  return resultDate;
}

export function addToDate(startDate: string, howMany: number, timeUnit: dayjs.OpUnitType): string {
  const date = dayjs(startDate + 'T00:00:00.000Z');
  return date.add(howMany, timeUnit).format('YYYY-MM-DD');
}

export function formatDate(dateISO: string, formatString: string): string {
  const date = dayjs(dateISO + 'T00:00:00.000Z');
  return date.format(formatString);
}
