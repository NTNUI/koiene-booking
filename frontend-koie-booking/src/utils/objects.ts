export const clone = (object: any) => Object.assign({}, object);

export const renameKey = (object: any, key: string, newKey: string): any => {
  const value = object[key];
  delete object[key];
  object[newKey] = value;
  return object;
};

export const sortObjectByKey = (unordered: any) => {
  let ordered: any;
  ordered = {};
  Object.keys(unordered)
    .sort()
    .forEach(function(key) {
      ordered[key] = unordered[key];
    });
  return ordered;
};
