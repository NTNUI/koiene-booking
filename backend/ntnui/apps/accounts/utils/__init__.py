from accounts.utils.mail_utils import send_gsuite_confirmation_mail
