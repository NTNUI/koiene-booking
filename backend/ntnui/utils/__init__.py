from ntnui.utils.pagination import PageSize7, PageSize20, PageSize100
