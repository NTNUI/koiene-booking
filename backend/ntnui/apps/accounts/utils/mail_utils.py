from django.core.mail import send_mail
from django.template.loader import render_to_string

from ntnui.enums import MembershipType


def new_membership(membership):
    """ Sends confirmation email to users and guests who signs up for events and sub-events. """

    # Mail header.
    sender = "noreply@mg.ntnui.no"
    receiver = [membership.member.email]
    subject = "Welcome"

    context = {"user": membership.member, "group": membership.group}

    # Mail body.
    plain_message = render_to_string("emails/new_membership.txt", context)
    html_message = render_to_string("emails/new_membership.html", context)

    send_mail(subject, plain_message, sender, receiver, html_message=html_message)


def send_gsuite_confirmation_mail(user, contact_email, password):
    # Mail header.
    sender = "noreply@mg.ntnui.no"
    receiver = [user.email]
    subject = "Opprettelse av ny NTNUI-bruker"

    context = {
        "first_name": user.first_name,
        "email": user.email,
        "contact_email": contact_email,
        "password": password,
    }

    # Mail body.
    plain_message = render_to_string("emails/new_gsuite_account.txt", context)
    html_message = render_to_string("emails/new_gsuite_account.html", context)

    send_mail(subject, plain_message, sender, receiver, html_message=html_message)


def send_welcome_mail_to_leader(user, group):
    """ Send a mail to a new leader """
    sender = "noreply@mg.ntnui.no"
    receiver = [user.email]
    subject = f"Velkommen som ny leder i {group}"

    context = {
        "first_name": user.first_name,
        "email": user.get_contact_email(),
        "group": group.name,
    }

    plain_message = render_to_string(
        template_name="emails/new_leader_membership.txt", context=context
    )
    html_message = render_to_string(
        template_name="emails/new_leader_membership.html", context=context
    )

    send_mail(
        subject=subject,
        message=plain_message,
        from_email=sender,
        recipient_list=receiver,
        html_message=html_message,
    )


def send_welcome_mail_to_cashier(user, group):
    """Send a mail to the new cashier"""
    sender = "noreply@mg.ntnui.no"
    receiver = [user.email]
    subject = f"Velkommen som ny kasserer i {group}"

    context = {
        "first_name": user.first_name,
        "email": user.get_contact_email(),
        "group": group.name,
    }

    plain_message = render_to_string(
        template_name="emails/new_cashier_membership.txt", context=context
    )
    html_message = render_to_string(
        template_name="emails/new_cashier_membership.html", context=context
    )

    send_mail(
        subject=subject,
        message=plain_message,
        from_email=sender,
        recipient_list=receiver,
        html_message=html_message,
    )


def send_welcome_mail_to_board_member(user, group):
    """Send a mail to the new cashier"""
    sender = "noreply@mg.ntnui.no"
    receiver = [user.email]
    subject = f"Velkommen som ny tillitsvalgt i {group}"

    context = {
        "first_name": user.first_name,
        "email": user.get_contact_email(),
        "group": group.name,
    }

    plain_message = render_to_string(
        template_name="emails/new_board_membership.txt", context=context
    )
    html_message = render_to_string(
        template_name="emails/new_board_membership.html", context=context
    )

    send_mail(
        subject=subject,
        message=plain_message,
        from_email=sender,
        recipient_list=receiver,
        html_message=html_message,
    )


def send_welcome_mail(membership):
    """ Delegate welcome mail to different board members """
    send_welcome_mail_to_board_member(user=membership.member, group=membership.group)
    if membership.type == MembershipType.leader:
        send_welcome_mail_to_leader(user=membership.member, group=membership.group)

    if membership.type == MembershipType.cashier:
        send_welcome_mail_to_cashier(user=membership.member, group=membership.group)
