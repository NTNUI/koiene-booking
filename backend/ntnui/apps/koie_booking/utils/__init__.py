from koie_booking.utils.mail_utils import send_confirmation_mail, send_koie_information_mail
from koie_booking.utils.date_utils import get_daterange
