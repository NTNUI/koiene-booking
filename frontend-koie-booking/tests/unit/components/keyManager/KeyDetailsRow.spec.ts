import Vue from 'vue';
import Vuetify from 'vuetify';
import { createShallowWrapper } from '../../utils';
import KeyDetailsRow from '@/components/keyManager/KeyDetailsRow.vue';
import { getKeyPickUps } from '../../__mocks__/keys';

Vue.use(Vuetify);

describe('Component KeyDetailsRow', () => {
  const wrapperOptions = {
    propsData: {
      keyDetail: Object.values(getKeyPickUps())[0],
      isPickUp: false,
    },
  };

  it('Matches snapshot', () => {
    const wrapper = createShallowWrapper(KeyDetailsRow, wrapperOptions);
    expect(wrapper).toMatchSnapshot();
  });

  it('Handles no props without crashing', () => {
    const wrapper = createShallowWrapper(KeyDetailsRow);
    expect(wrapper.exists()).toBeTruthy();
  });

  it('Renders PickUp field if pickup', () => {
    wrapperOptions.propsData.isPickUp = true;
    const wrapper = createShallowWrapper(KeyDetailsRow, wrapperOptions);
    expect(wrapper.find({ ref: 'pickUpStartDate' }).exists()).toBeTruthy();
  });

  it('getColorFn returns white if no props are provided', () => {
    const wrapper = createShallowWrapper(KeyDetailsRow);
    expect(wrapper.vm.getColorFn()).toEqual({ color: 'white' });
  });
});
