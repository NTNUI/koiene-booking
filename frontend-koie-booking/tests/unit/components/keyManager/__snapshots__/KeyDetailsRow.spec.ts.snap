// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component KeyDetailsRow Matches snapshot 1`] = `
<tr style="background-color: rgb(34, 34, 34);">
  <td>
    ola1@normann.com
  </td>
  <td>
    Mevasskoia
  </td>
  <td><span style="color: white;">2020-10-29
    </span></td>
  <td><span> 2020-11-01 </span></td>
  <td></td>
</tr>
`;
